workspaces/global-header/plugins/global-header/src/components/HeaderDropdownComponent/ProfileDropdown.tsx

```diff
@@ -40,9 +37,8 @@ export interface ProfileDropdownProps {
 
 export const ProfileDropdown = ({ layout }: ProfileDropdownProps) => {
   const { anchorEl, handleOpen, handleClose } = useDropdownManager();
+  const { backstageIdentity } = useUserProfile();
```

Comment:
Just want confirm or discuss because we want backport this:

1. `useUserProfile` returns also a `displayName`. Would it be the same result? Is it maybe simpler? See https://github.com/backstage/backstage/blob/master/plugins/user-settings/src/components/useUserProfileInfo.ts#L75
2. Can we use `profile.picture` to show the profile Avatar and callback to the existing `AccountCircleOutlinedIcon`? See https://github.com/backstage/backstage/blob/master/plugins/user-settings/src/components/General/UserSettingsSignInAvatar.tsx -- We can do that in another PR maybe.

Reply:
When logged in as Guest, the displayName returns "user:development/guest" instead of "development/guest" like how it shows up in the Settings Page.
Similarly for OIDC user, it returns "rhdh-qe" instead of the "RHDH QE"

Reply:
Created PR https://github.com/redhat-developer/rhdh-plugins/pull/737/files for adding the Profile picture